config/manifests/manifest_common.json

```diff
@@ -512,8 +512,8 @@
     },
     {
       "matches": [
-        "*://steamcommunity.com/sharedfiles/editguide?id=*",
-        "*://steamcommunity.com/sharedfiles/editguide/?id=*"
+        "*://steamcommunity.com/sharedfiles/editguide?*appid=*",
+        "*://steamcommunity.com/sharedfiles/editguide/?*appid=*"
```

Comment:
This is not good. It won't load the guide features without the `appid` parameter! When I click edit guide, it only had a `id` parameter. So this change is wrong!

E.g.
1. https://steamcommunity.com/id/revadike/myworkshopfiles/?section=guides
2. https://steamcommunity.com/sharedfiles/filedetails/?id=2168019711
3. I click: Edit basic guide information
4. https://steamcommunity.com/sharedfiles/editguide/?id=2168019711

Reply:
I was wondering how such a mistake could've happened while merging but when I created a guide I also only got the appid parameter.
Does it even make sense to match the query parameters or are there pages where we don't wanna run without those params?

Reply:
I think it's fine without query parameters. It does have this: 
https://github.com/tfedor/AugmentedSteam/blob/develop/src/js/Content/Features/Community/EditGuide/FCustomTags.js#L64

But using "recent" as id when its missing should handle it correctly, I think.

Reply:
oops, so it originally worked while editing saved guides only, in that case the match should be `"*://steamcommunity.com/sharedfiles/editguide/?*"`

> Does it even make sense to match the query parameters or are there pages where we don't wanna run without those params?

Yes because without query params the link just redirects to workshop home.

Reply:
Well then what's the problem? If you can't load the page without query params anyway...